addons/account_edi_facturx/models/account_edi_format.py

```diff
@@ -190,20 +193,21 @@ def _import_facturx(self, tree, invoice):
         # self could be a single record (editing) or be empty (new).
         with Form(invoice.with_context(default_move_type=default_move_type,
                                        account_predictive_bills_disable_prediction=True)) as invoice_form:
-            # Partner (first step to avoid warning 'Warning! You must first select a partner.').
+            self_ctx = self.with_context(allowed_company_ids=[invoice_form.journal_id.company_id.id])
```

Comment:
use `with_company(invoice.company_id)` (or `force_company_id` if defined after 14.0)

Reply:
`with_company` will only prefix the list of `allowed_company_ids` with the new id, but other ids are still valid.
Anyway, if I cannot access `allowed_company_ids` I should use another method.